### ISSUE TITLE
Connection-related messages in channels or queries

### ISSUE THREAD

**Issue by djasa** [SrainApp/srain#377]
In srain's "bubbly" design of chat widgets, connection-related messages can take huge amount of space that should be used by actual user's messages. The logical space for these messages is server tab and in channels/queries, it makes sense to merge them into a single bubble telling user since when they're disconnected and if there's connection attempt ongoing or if srain is waiting.

(Perfect would be to handle unstable connection and merge more disconnection infos with no actual mesages in between to a single bubble showing something like this: [disconnected 2022-08-30 19:00-2022-08-31 9:00, 9:37-9:55, 15:50-17:25], but if that's too much work, don't let perfect be enemy of the good. 🙂️)

### version
```
$ flatpak info im.srain.Srain

Srain - Modern IRC client written in GTK

          ID: im.srain.Srain
         Ref: app/im.srain.Srain/x86_64/stable
        Arch: x86_64
      Branch: stable
     Version: 1.4.0
     License: GPL-3.0
      Origin: flathub
  Collection: org.flathub.Stable
Installation: system
   Installed: 915,5 kB
     Runtime: org.gnome.Platform/x86_64/42
         Sdk: org.gnome.Sdk/x86_64/42

      Commit: 2055941c2b9d402c1c383973df7afe8f511fd439484420746024afe9e5539f0d
      Parent: 7db333a933b6921fe1a5ebf850fc1e195f2378e9ee0e878b3a6134f2e12a3c2a
     Subject: Update to 1.4.1 (653ddbd7)
        Date: 2022-08-19 16:23:22 +0000
```

**Comment by SilverRainZ** [comment]
Yes it makes sense. Unfortunately, the bubble chat widget hasn't enough state to aggregate these connection-related messages.

For now, https://srain.silverrainz.me/commands.html#filter-unfilter may be helpful.

**Comment by djasa** [comment]
Would it be possible to at least aggregate server messages coming in a row into a single bubble like it is done for user messages? Even that would help tremendously.

**Comment by SilverRainZ** [comment]
> Would it be possible to at least aggregate server messages coming in a row into a single bubble like it is done for user messages? Even that would help tremendously.

It is possible, but not so beautiful, coming in a row breaks the text-centered style.
